addons/mrp/models/mrp_unbuild.py

```diff
@@ -121,14 +121,23 @@ def action_unbuild(self):
         for produce_move in produce_moves:
             if produce_move.has_tracking != 'none':
                 original = original_quants.filtered(lambda quant: quant.product_id == produce_move.product_id)
-                if original and not original[0].lot_id:
```

Comment:
In theory you could have consumed one less in the MO, but not sure if we should even handle it.

Reply:
Ok let handle it by not raising if we can't restore the totality but shoud do less (discussed with jco)